py3-setuptools-73.yaml

```diff
@@ -0,0 +1,72 @@
+package:
+  name: py3-setuptools-73
```

Comment:
Maybe a dumb question, if this is a version stream, should it have a `provides` of `py3-setuptools=73` or something like that?

Reply:
Not a dumb question.
We could do that. I think the answer is no, because we do not really want to have this package used.  I would hope we can drop it as soon as gobject-introspection changes.

Reply:
Ah okay, got it! What would you think of putting a comment somewhere toward the top here to explain that? Like "hey, you might want to version-stream-ize this, but don't! we want to get rid of this package as soon as X....."

Reply:
@luhring ... good news. by waiting, now we have a better fix.

Reply:
Yay!